app/src/main/res/values/strings.xml

```diff
@@ -114,58 +111,54 @@ Reminder: Easer is still in development. It may contain bugs and the UI definite
-    <string name="pref_use_root_summary">Use root permission for specific functions (plugins), which may help resolve compatibility issue on some devices. Root permission won\'t be used elsewhere.</string>
+    <string name="pref_use_root_summary">Use root permission for specific functions (plugins) to help resolve compatibility issues on some devices.</string>
 
-    <string name="pref_cooldown_title">Cool down time</string>
-    <string name="pref_cooldown_summary">Minimal interval between two triggers of the same event</string>
-    <string name="cooldown_time_illformed">The number must not be lower than 0 and not too large to fit in a integer</string>
+    <string name="pref_cooldown_title">Event only once</string>
+    <string name="pref_cooldown_summary">Timeframe in seconds to treat two occurances as one event.</string>
+    <string name="cooldown_time_illformed">Your number is too high, or not a positive whole number.</string>
 
     <string name="pref_passive_mode_title">Passive mode</string>
-    <string name="pref_passive_mode_summary_on">Listen to Event; no checking when activated</string>
-    <string name="pref_passive_mode_summary_off">Check the satisfaction state when (each) Event is activated</string>
+    <string name="pref_passive_mode_summary_on">Don\'t trigger an active event again.</string>
+    <string name="pref_passive_mode_summary_off">Check if the event should be triggered again when (each) event is activated.</string>
 
-    <string name="pref_enabled_plugins_title">Enabled plugins</string>
-    <string name="pref_enabled_plugins_summary">Enable or disable certain plugins</string>
+    <string name="pref_enabled_plugins_title">Activated plugins</string>
+    <string name="pref_enabled_plugins_summary">Turn on or off certain plugins</string>
 
     <string name="pref_use_12_hour_clock_title">Use 12-hour clock</string>
 
     <string name="pref_export_title">Export</string>
-    <string name="pref_export_summary">Export all events and profiles to SD Card.\nFile will be named after current time.</string>
+    <string name="pref_export_summary">Export all events and profiles to SD card.\nThe current time as of saving will be used as filename.</string>
     <string name="pref_import_title">Import</string>
-    <string name="pref_import_summary">Import all events and profiles in a zip file.\nExisting events and profiles with same name will be replaced.</string>
+    <string name="pref_import_summary">Import all events and profiles in a ZIP file.\nDuplicated event and profile names will be replaced.</string>
 
     <string name="pref_convert_data_title">Convert data</string>
     <string name="pref_convert_data_summary">Convert all data to the recommended format</string>
 
-    <string name="message_plugin_not_compatible">This plugin is not compatible with this device.</string>
+    <string name="message_plugin_not_compatible">This device can not run this plugin.</string>
```

Comment:
Actually the compatibility checking is more than this... Though currently it only checks hardware... Just in case there is a better description, or I'll agree with you.

Reply:
So is it a hardware or software error? ~"Buy a device with hardfloat to use this plugin", or ~"Upgrade to Android 6.0 or later to use this plugin" would be helpful.